.github/workflows/scripts/linux_arm64/rh8/build_llvm.sh

```diff
@@ -43,7 +43,7 @@ cmake -S /catalyst/mlir/llvm-project/llvm -B /catalyst/llvm-build -G Ninja \
     -DLLVM_INSTALL_UTILS=ON \
     -DLLVM_ENABLE_ZLIB=OFF \
     -DLLVM_ENABLE_ZSTD=FORCE_ON \
-    -DLLVM_ENABLE_LLD=OFF \
+    -DLLVM_ENABLE_LLD=ON \
     -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
     -DPython3_EXECUTABLE=/usr/bin/python3 \
     -DPython3_NumPy_INCLUDE_DIRS=/opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/lib/python${PYTHON_VERSION}/site-packages/numpy/core/include \
```

Comment:
Should LLVM not also be built with protected symbols? (next line)

Reply:
Yes. I 've added it.